extra/mariabackup/fil_cur.cc

```diff
@@ -253,7 +253,7 @@ xb_fil_cur_open(
 	if (!node->space->crypt_data
 	    && os_file_read(IORequestRead,
 			    node->handle, cursor->buf, 0,
-			    page_size.physical())) {
+			    page_size.physical()) == DB_SUCCESS) {
```

Comment:
This is not covered by tests.

Reply:
This is something that was added recently as part of MDEV-12112, MDEV-18129, MDEV-18025. This could be dead code unless the option `--skip-encrypted-backup` is present. With the default value of that option we are working around MDEV-18128.